platform/fabricx/core/finality/nlm_test.go

```diff
@@ -17,12 +17,16 @@ import (
 	"github.com/hyperledger-labs/fabric-smart-client/platform/common/driver"
 	"github.com/hyperledger-labs/fabric-smart-client/platform/fabric"
 	fdriver "github.com/hyperledger-labs/fabric-smart-client/platform/fabric/driver"
-	"github.com/hyperledger-labs/fabric-smart-client/platform/fabricx/core/finality/fakes"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/fabricx/core/finality/mock"
 	"github.com/hyperledger/fabric-x-committer/api/protoblocktx"
 	"github.com/hyperledger/fabric-x-committer/api/protonotify"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
+// To re-generate the "./mock/fake_notifier_open_notification_stream_client.go" 
+// add this line: "//go:generate go tool counterfeiter . Notifier_OpenNotificationStreamClient" 
+// above Notifier_OpenNotificationStreamClient interface in fabric-x-committer/api/protonotify/notify_grpc.pb.go
+// and run "go generate" 
```

Comment:
Can't we just add `//go:generate go tool counterfeiter . Notifier_OpenNotificationStreamClien` here? so we can generate it without modifying this line?